support/sdk-test/build_var.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -e
+
+mkdir -p ~/avocado-qemux86-64/rootfs/var
+
+dnf -y install avocado-extension-debug-tweaks \
+    --setopt=tsflags=noscripts \
+    --installroot ~/avocado-qemux86-64/rootfs/
+
+rm -f /images/avocado-qemux86-64/avocado-image-var-avocado-qemux86-64.btrfs
+
+mkfs.btrfs -r ~/avocado-qemux86-64/rootfs/var \
```

Comment:
Oh this is interesting. very neat

Reply:
Thank god this `--subvol` feature exists. And thank you for patching `btrfs-tools` so I can use it.